components/script/dom/element.rs

```diff
@@ -286,9 +300,21 @@ impl Element {
             custom_element_reaction_queue: Default::default(),
             custom_element_definition: Default::default(),
             custom_element_state: Cell::new(CustomElementState::Uncustomized),
+            done_creating: Cell::new(false),
+        }
+    }
+
+    pub fn done_creating(&self) {
+        self.done_creating.set(true);
+        if let Some(el) = self.downcast::<HTMLInputElement>() {
+            el.refresh_value(el.Value());
```

Comment:
Can we avoid passing the element's own value to `refresh_value`? This seems like that involves cloning the value for no reason.